nats/replica_client.go

```diff
@@ -0,0 +1,402 @@
+package nats
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/benbjohnson/litestream"
+	"github.com/benbjohnson/litestream/internal"
+	natsClient "github.com/nats-io/nats.go"
```

Comment:
Why not use `nats` as the alias since that's what's used in the client package? https://github.com/nats-io/nats.go/blob/v1.37.0/context.go#L14

Reply:
because I was in the `nats` package I thought that would cause issues, apparently not the case, fixed